package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "lss-manager-v4",
-    "version": "4.7.7+20230815.1041",
+    "version": "4.7.7+20230819.2317",
```

Comment:
Something doesn't add up. Either PR title is wrong or the version :D

Reply:
=> 2ef79d1 :sweat_smile: